### ISSUE TITLE
Telegram Notifications only work for new libraries.

### ISSUE THREAD

**Issue by majorpdd** [Tautulli/Tautulli#1123]
When starting to play through Plex it seems if i create a new library then notifications on that library seem to work, but not all my existing ones which did before the new commit.  Test notification also works.

**Version:**
1.4.21
**Branch:**
v2
**Commit hash:**
845c3d419a2c321aae2feaa1121fa12ea1c3c3f8
**Operating system:**
Server 2012 R2
**Python version:**
2.7.12
**What you did?**
Updated to new commit
**What happened?**
Since then it stopped working.
**What you expected?**
Expected notification to Telegram which was working and now it does not.

**Link to logs:**
[plexpy.log](https://github.com/JonnyWong16/plexpy/files/1384607/plexpy.log)

**Comment by JonnyWong16** [comment]
Thanks for the report, but I am not supporting v2 right now since it's not released. Most likely your configuration is wrong somewhere.

**Comment by majorpdd** [comment]
Nope... Moved the database back to Master and exactly the same issue.

**Comment by JonnyWong16** [comment]
The database is not backwards compatible. I cannot guarantee that a v2 database will work with v1.

**Comment by majorpdd** [comment]
I got used to not having notifications, but it stopped logging after a recent Plex update so I had to fetch a new Plex token to fix...

Thought I would have another go at fixing this! but cannot get anywhere!  I have even tried the newest version then moving my database across but get Internal 500 issue, even without my database, blank new dl and Internal 500 Error.  I know you don't support v2 and I know you get asked all the time... so i'm not going to ask, but a pointer in the right direction? I have even checked for stuck notifications in the db file, nothing there.  It must be in the DB file.. :(
